test/misc.jl

```diff
@@ -522,6 +524,15 @@ if stdout isa Base.TTY
     @test_throws KeyError stdout[:bar]
 end
 
+@testset "`displaysize` on closed TTY #34620" begin
+    Main.FakePTYs.with_fake_pty() do rawfd, _
+        tty = Base.TTY(rawfd)
```

Comment:
```suggestion
        tty = open(rawfd)::Base.TTY
```

(though there's also a typo on L292 `s/pipe/io` you'll need to fix)